Files/ViewModels/CurrentInstanceViewModel.cs

```diff
@@ -11,11 +12,14 @@ public class CurrentInstanceViewModel : ObservableObject
          * values being manipulated inside the setter blocks.
          */
 
+        public IJsonSettings JsonSettings { get; }
+
         public FolderSettingsViewModel FolderSettings { get; }
 
         public CurrentInstanceViewModel(IShellPage associatedInstance)
         {
             FolderSettings = new FolderSettingsViewModel(associatedInstance);
+            JsonSettings = new JsonSettingsViewModel(associatedInstance);
```

Comment:
Should this really be tied to the current instance? 
@gave92 Any thoughts on this?

Reply:
Doesn't look like JsonSettingsViewModel uses associatedInstance, so no need to keep *another* reference to IShellPage :)

Reply:
Where should I put it?

Reply:
Simply meant that there is not point in passing associatedInstance to JsonSettingsViewModel which doesn't use it